### DISCUSSION TITLE
Running Prisma ^5.15.0 in Node js 14.10.1 environment

### DISCUSSION THREAD

**Discussion by interk0t** [prisma/prisma#25664] [Q&A]
### Question

I have a specific server that runs on a **Node** version **14.10.1** and I can't influence it in any way. In this project I use **Prisma** and I can only use version **4.3.1** because it works fine in the environment of the **Node 14.10.1**, but in **4.3.1** there is no **previewFeatures = ["prismaSchemaFolder"]** and I really need it, so I need to install at least **5.15.0** version of **Prisma**, and here the problems begin.

If I install **5.15.0 Prisma**, I get an error on the server:
`Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './runtime/library.js' is not defined by "exports" in D:\frontend\RAGEMP\server-files\node_modules\.pnpm\@prisma+client@5.15.0_prisma@5.15.0\node_modules\@prisma\client\package.json`

or there may be such an error, depending on the build

`SyntaxError: Unexpected token '??='`

which indicates that this is a newer syntax for **Node** version **14.10.1** and it cannot work with it
if I transpile the code through babel, the project will not build at all with an error:

`src/index.ts → ../packages/server/index.js...
[!] (plugin rpt2) RollupError: [plugin rpt2] ../node_modules/.pnpm/@prisma+client@5.15.0_prisma@5.15.0/node_modules/@prisma/client/default.js: Unknown version 14.10.1 of Node.js
../node_modules/.pnpm/@prisma+client@5.15.0_prisma@5.15.0/node_modules/@prisma/client/default.js`

Question: how to make **Prisma** version **^5.15.0** work on **Node**  version **14.10.1**?

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @interk0t 

From Prisma ORM version 5.0.0, the minimum supported version of Node.js is 16.13.0. If your project uses an earlier version, you will need to upgrade it. This is also stated in the [documentation](https://www.prisma.io/docs/orm/more/upgrade-guides/upgrading-versions/upgrading-to-prisma-5#nodejs-minimum-version-change) for the upgrade guide.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
